refactor: relaxes requirements versions & more

fsologureng opened the pull request:
- removes version restrictions of dependencies.
 - bump minor version.
 - removes win-like EOL & EOF.

catalinaIlabaca reviewed (approved):
Deberías agregar que se modifico el formato.

fsologureng commented:
> Deberías agregar que se modifico el formato.

lo del salto de linea? ya está declarado...

fsologureng committed:
refactor: remove version constraints in requirements & more
     - removes version constraints in requirements.
     - bump minor version.
     - removes win-like EOL & EOF.

catalinaIlabaca commented on requirements.in:
En el setup.py en install_requires se menciona unidecode, ¿será necesario agregalo en este archivo?
